pkg/file/builder_test.go

```diff
@@ -389,10 +390,29 @@ func existingFilterChainState() *state.KongState {
 	return s
 }
 
+func existingDegraphqlRouteState() *state.KongState {
+	s, _ := state.NewKongState()
```

Comment:
Why not check the error? I'd suggest we consider adding a `*testing.T` param and `require.NoError(t, err)` here (as well as `t.Helper()` to make sure the correct call site of `existingDegraphqlRouteState` is printed in the logs). WDYT?